pom.xml

```diff
@@ -90,6 +90,12 @@
     </build>
 
     <dependencies>
+        <dependency>
+            <groupId>org.jenkins-ci.lib</groupId>
+            <artifactId>envinject-lib</artifactId>
+            <version>1.27</version>
+        </dependency>
```

Comment:
Why do we need to add this for this PR. To the best of my knowledge, the dependency was [recently replaced](https://github.com/jenkinsci/envinject-plugin/commit/af113cfd5008e73b349266891b89d6b043296816) with `envinject-api`.

Reply:
This PR does not contribute any and master appears to be working without it. Putting aside it should be made a test-scoped dependency if this is truly needed for a test.